Dependencies.props

```diff
@@ -44,6 +45,7 @@
   </PropertyGroup>
   <PropertyGroup Label="ServiceFabric packages for Omex Extensions">
     <Microsoft_ServiceFabric_Version>7.2.434</Microsoft_ServiceFabric_Version>
+    <Microsoft_ServiceFabric_Client_Http_Version>4.2.0</Microsoft_ServiceFabric_Client_Http_Version>
```

Comment:
You might be able to start with the latest version - 4.4.0 :)

Reply:
how exciting!